src/styles/blocks/card.scss

```diff
@@ -0,0 +1,118 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: $card-width;
+  padding: 32px 16px 16px;
+
+  box-sizing: border-box;
+
+  background: $white-color;
+  border: 1px solid #f2f2f2;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin-top: 40px;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+
+    color: $dark-color;
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $secondary-color;
+  }
+
+  &__raiting {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    line-height: 18px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    line-height: 18px;
+    color: $dark-color;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $dark-color;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    font-weight: 400;
+    color: $secondary-color;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+  }
+
+  &__code,
+  &__price,
+  &__btn-buy {
+    margin-bottom: 16px;
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    padding: 11px;
+    box-sizing: border-box;
+
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    color: $white-color;
+    text-transform: uppercase;
+
+    background-color: $primary-color;
+    border: 1px solid transparent;
+    border-radius: 5px;
+  }
+
+  &__buy-button:hover {
+    color: $primary-color;
+
+    background-color: $white-color;
+    border: 1px solid $primary-color;
+    border-radius: 5px;
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 10th point